agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java

```diff
@@ -169,6 +173,9 @@ static void setExtraAttributes(AbstractTelemetryBuilder telemetryBuilder, Attrib
     attributes.forEach(
         (key, value) -> {
           String stringKey = key.getKey();
+          if (stringKey.startsWith("applicationinsights.internal.")) {
```

Comment:
when do we officially deprecated internal attributes?

Reply:
we don't need to deprecate `internal` attributes, they are just internal implementation details that we don't expose to customers